core/src/main/resources/schema/infinispan-config-15.0.xsd

```diff
@@ -357,6 +357,7 @@
             <xs:documentation>Defines a DIST_* mode cache configuration.</xs:documentation>
           </xs:annotation>
         </xs:element>
+        <xs:any namespace="##other"/>
```

Comment:
Do we expect a custom namespace for a different cache type?

Reply:
I think this is a leftover of the grander plan I had for cache aliases. I'll revert this